packages/non-core/coffeescript-compiler/package.js

```diff
@@ -13,7 +13,7 @@ Package.describe({
   summary: 'Compiler for CoffeeScript code, supporting the coffeescript package',
   // This version of NPM `coffeescript` module, with _1, _2 etc.
   // If you change this, make sure to also update ../coffeescript/package.js to match.
-  version: '2.0.3_2'
+  version: '2.0.3-2-beta161.13'
```

Comment:
The `-beta161.n` suffixes can be reverted for `coffeescript*` packages once `babel-compiler@7.0.0` is published with the Meteor 1.6.1 release. In the meantime, the suffixes are necessary so that folks can use `coffeescript` with the Meteor 1.6.1 betas.